lib/r_tree.dart

```diff
@@ -28,6 +28,8 @@ library r_tree;
 
 import 'dart:math';
 
+import 'package:r_tree/src/r_tree/quickselect.dart';
```

Comment:
Should we be consistent with `part` directives?

What about if we removed that directive altogether and be explicit about what we import and export?  I don't think that will matter much with this small of a library though.

Reply:
I'll make it a part for now to be consistent